docs/JSON-RPC-API/JSON-RPC-API.md

```diff
@@ -0,0 +1,17 @@
+description: Pantheon JSON-RPC API reference
+<!--- END of page meta data -->
+
+# JSON-RPC API Overview
+
+The Pantheon JSON-RPC API uses the [JSON](http://json.org/) (RFC 4627) data format, which can represent 
+objects and data fields as collections of name/value pairs, in a relatively readable, hierarchical form. 
+Values have specific data types such as quantities (decimal integers, hexadecimal numbers, strings) and 
+unformatted data (byte arrays, account addresses, hashes, and bytecode arrays).
+
+RPC is the remote procedure call protocol (RFC 1831), which is stateless and transport agnostic in that the concepts 
```

Comment:
We should also mention the JSON-RPC 2.0 spec that we use.

https://www.jsonrpc.org/

Reply:
Done